plugins/aea-cli-ipfs/aea_cli_ipfs/registry.py

```diff
@@ -152,34 +151,17 @@ def register_item_to_local_registry(
 
 
 def fetch_ipfs(
-    item_type: str,
     public_id: PublicId,
     dest: str,
-    remote: bool = True,
```

Comment:
So I removed the `remote` argument because this was useless any way since we've stopped using local IPFS nodes and local IPFS registry.